Source/Core/AudioCommon/SurroundDecoder.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <memory>
+
+#include "Common/CommonTypes.h"
+#include "Common/FixedSizeQueue.h"
+
+class DPL2FSDecoder;
+
+namespace AudioCommon
+{
+class SurroundDecoder
+{
+public:
+  explicit SurroundDecoder(uint32_t sample_rate, uint32_t frame_block_size);
```

Comment:
We generally use our `u32` type alias instead of `uint32_t` (ditto for any other `<s|u>[size]_t` types)